pkg/tasks/delete_snapshots.go

```diff
@@ -178,9 +180,34 @@ func (ds *DeleteSnapshots) deleteOrUpdatePulpContent(snap models.Snapshot, repo
 
 	deleteVersionHref, err := ds.getPulpClient().DeleteRpmRepositoryVersion(ds.ctx, snap.VersionHref)
 	if err != nil {
+		distributions, err := ds.getPulpClient().ListDistributions(ds.ctx, ds.domainName)
+		if err != nil {
+			return err
+		}
+		publicationDistMap := make(map[string][]string)
+		logger.Debug().Msgf("Distribution count: %d", len(*distributions))
+		for i, dist := range *distributions {
+			logger.Debug().Msgf("Distribution #%d", i+1)
+			logger.Debug().Msgf("Distribution href: %v", *dist.PulpHref)
+			logger.Debug().Msgf("Distribution base path: %v", dist.BasePath)
```

Comment:
I think we only want to print this stuff for distributions that are including the publication we're deleting (via the version delete).  Otherwise it would print a LOT of output!

Reply:
yep was thinking the same, it was a lot even locally :laughing: so only log if the dist.publication matches the snapshot's publication href or version href? the terms are confusing me

Reply:
does this make more sense?

```
logger.Debug().Msgf("Checking distributions for publication: %v", snap.PublicationHref)
for _, dist := range *distributions {
	if dist.Publication.IsSet() {
		pub := dist.Publication.Get()
		if *pub == snap.PublicationHref {
			logger.Debug().Msg("Found distribution using deleted publication")
			logger.Debug().Msgf("  - pulp href: %v", *dist.PulpHref)
			logger.Debug().Msgf("  - base path: %v", dist.BasePath)
			logger.Debug().Msgf("  - publication: %v", *pub)
		}
	}
}
```

Reply:
I think thats right.  You might ensure pub isn't nil there, but otherwise looks right